app/src/components/main/scrollWrapper/scrollWrapper.jsx

```diff
@@ -141,6 +142,7 @@ export class ScrollWrapper extends Component {
     if (scrollLeft !== lastViewScrollLeft) {
       this.scrollbars.thumbHorizontal.style.opacity = 1;
     }
+    forceCheck();
```

Comment:
Maybe we can somehow utilize `onLazyLoad` function? I mean, move this check there or inside the condition on line 139. What do you think?